code-challenges.test.js

```diff
@@ -1,51 +1,127 @@
 // ASSESSMENT 4: JavaScript Coding Practical Questions with Jest
 
-// Please read all questions thoroughly
-// Pseudo coding is REQUIRED
-// If you get stuck, please leave comments to help us understand your thought process
-
-// Use test driven development to complete the following questions
-// Add appropriate dependencies: $ yarn add jest
+// --------------------1) Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
 
-// Reminder: The test will call your function
-// Run the file with the following command: $ yarn jest
+// Pseudo Code:
+// Input: an array
+// Output: a shuffled array with the first item from the array removed
+// I need to make a copy of the original array using the slice method so that I can shuffle the copy without changing the original array
+// I would need to use the shift method to remove the first element from the copied array.
+// I may need to use the sort and math.random methods to shuffle the elements randomly.
+// I learned that 0.5 is used as a threshold to determine the order of the shuffled elements. A value greater than 0.5 will sort the elements in ascending order, whereas a value less than 0.5 will sort the elements in descending order. I can't use a number too close to 0 or 1 either since it will result in less randomness.
```

Comment:
0.5 is used because Math.random() on its own gives a number between 0 and 1. Adding the - 0.5 to the end makes the output evenly randomized to a negative or positive outcome, which is used by the comparison function in .sort to effectively shuffle the items in the array.